spec/fare_transactions.schema.json

```diff
@@ -78,10 +78,18 @@
       "description": "Identifies the unique stop-path for a trip, may be distinct from GTFS shapes.shape_id"
     },
     {
-      "name": "stop_sequence",
+      "name": "trip_stop_sequence",
+      "type": "integer",
+      "description": "The temporal sequence of this transaction's enclosing stop visit, if any, within its performed trip, starting at 1 and containing no gaps. This may differ from scheduled_stop_sequence.",
```

Comment:
I find the description a bit unclear. Maybe something more akin to the GTFS stop_sequence language: 
```
"The actual order of stops visited within a performed trip. The values must start at 1 and must be consecutive along the trip.<br/> Example: A bus departs the first stop and detours around the second and third scheduled stops, visiting one unscheduled stop and resuming regular service at the 4th scheduled stop. The `scheduled_stop_sequence` is [1, null, 4], and the `trip_stop_sequence` is [1, 2, 3]."